building/upstream.log

```diff
@@ -34,3 +34,14 @@
   comment: mostly matches manual git-archive export, except for a couple of minor versioning differences, as expected
 
   compress: gunzip, xz
+
+- output: coreos_restructured-1478.0.0.cpio.gz
+  last: 2018-01-30
+
+  source: https://alpha.release.core-os.net/amd64-usr/1478.0.0/coreos_production_pxe_image.cpio.gz
+  signature: https://alpha.release.core-os.net/amd64-usr/1478.0.0/coreos_production_pxe_image.cpio.gz.sig
+
+  signing-key: 04127D0BFABEC8871FFB2CCE50E0885593D2DCB4
+  signing-key-retrieved-from: https://coreos.com/security/image-signing-key/
+
+  process: restrip-coreos.sh
```

Comment:
I'm not sure exactly how the `restrip-coreos.sh` script expects to receive its input.  Maybe make the instructions more explicit?

Reply:
I've added explanatory checks to the script. Does that help?